README.md

```diff
@@ -76,11 +81,19 @@ the actual driver's name.
           args:
             - "--csi-address=/csi/csi.sock"
             - "--kubelet-registration-path=/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock"
+            - "--health-port=9809"
           volumeMounts:
             - name: plugin-dir
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
+          ports:
+            - containerPort: 9809
+              name: healthz
+          livenessProbe:
```

Comment:
Should we set an initialDelay to give some time to create the socket? And should we set timeoutSeconds to give some grace period in case the system is very busy?

The other defaults seem reasonable to me